.circleci/config.yml

```diff
@@ -81,8 +81,7 @@ jobs:
                 name: "Collect Artifacts Published"
                 command: |
                   shopt -s globstar
-                  cp build-artifacts/release.yml ./publish-artifacts
-                  cp build-artifacts/*.jar ./publish-artifacts
+                  cp build-artifacts/* ./publish-artifacts
```

Comment:
I think the general rule is to only grab the release file and jars as other build artefacts don't need to be published - eg debug symbols. However, I don't think it's a big deal and we can review if needed.